drivers/npmx/npmx_shell.c

```diff
@@ -284,6 +321,11 @@ static void print_convert_error(const struct shell *shell, const char *src, cons
 	shell_error(shell, "Error: unable to convert %s to %s.", src, dst);
 }
 
+static void print_recalculate_error(const struct shell *shell, const char *temp)
```

Comment:
Will not be needed, when https://github.com/NordicSemiconductor/npmx-zephyr/pull/106/files is merged. Also, should probably be added in the first commit.